conf/keycloak/oidc-keycloak-auth-provider.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "oidc-keycloak",
+  "factoryAlias": "oidc",
+  "title": "OIDC-Keycloak",
+  "subtitle": "OIDC-Keycloak",
+  "factoryData": "type: oidc | issuer: http://localhost:8090/auth/realms/oidc-realm | clientId: oidc-client | clientSecret: ss6gE8mODCDfqesQaSG3gwUwZqZt547E",
```

Comment:
not so secret in github?

Reply:
This setup is intended for local testing only. For a remote or production OIDC provider, this configuration file, as well as the Realm file (if using Keycloak for OIDC) should not be included in the repository.

Reply:
It's fine. Dev only. Never prod, please. I added a warning about this: bb625ea

In the future, assuming we ship a Keycloak config for prod, can worry about secrets then.